src/configuration/GPG/GnuPG/params.txt

```diff
@@ -0,0 +1,9 @@
+Key-Type: RSA
+Key-Length: 8192  
+Subkey-Type: RSA
+Subkey-Length: 8192
```

Comment:
same as above. 4k is fine.

Reply:
Keys with more than 4096-bits also don't work properly on some gpg versions.